README.md

```diff
@@ -1 +1,4 @@
-# watchers
\ No newline at end of file
+# Notify watcher watchers
+
+Repository for all Notify watcher watchers.
+If you'd like to add a watcher submit a PR!
```

Comment:
(out of the scope of this PR, but we should eventually have a "contributing" section)

Reply:
yes, we should!